src/index.ts

```diff
@@ -0,0 +1,77 @@
+import Eth from 'ethjs';
+import registryMap from './registry-map.json';
+import abi from './abi.json';
+
+interface HttpProvider {
+  host: string;
+  timeout: number;
+}
+
+interface Provider {
+  network: string;
+  provider: HttpProvider;
+}
+
+interface DeployedRegistryContract {
+  entries (bytes: Uint8Array): unknown[];
+}
```

Comment:
@brad-decker, right now we're duplicating this interface in the manual declarations file and on these lines. We couldn't figure out a way to do dual exports from the `ethjs` declarations file without TypeScript or eslint crapping out. Do you have any idea on how to DRY this up?